src/integrations/bluetooth-low-energy/bluetooth-low-energy.config.ts

```diff
@@ -1,28 +1,67 @@
+import * as Joi from 'joi';
+import * as jf from 'joiful';
+
+class TagOverride {
+  @(jf.string().optional())
+  id?: string;
+  @(jf.string().optional())
+  name?: string;
+  @(jf.number().negative().optional())
+  measuredPower?: number;
+  @(jf.number().integer().optional())
+  batteryMask?: number;
+}
+
 export class BluetoothLowEnergyConfig {
+  @(jf.number().integer().min(0))
   hciDeviceId = 0;
+  @(jf.array({ elementClass: String }).optional())
   whitelist: string[] = [];
+  @(jf.boolean().optional())
   whitelistRegex = false;
+  @(jf.array({ elementClass: String }).optional())
   allowlist: string[] = [];
+  @(jf.boolean().optional())
   allowlistRegex = false;
+  @(jf.array({ elementClass: String }).optional())
   blacklist: string[] = [];
+  @(jf.boolean().optional())
   blacklistRegex = false;
+  @(jf.array({ elementClass: String }).optional())
   denylist: string[] = [];
+  @(jf.boolean().required())
   denylistRegex = false;
+  @(jf.boolean().required())
   processIBeacon = true;
+  @(jf.boolean().required())
   onlyIBeacon = false;
+  @(jf.number().integer().min(0).max(0xffff).required())
   majorMask = 0xffff;
+  @(jf.number().integer().min(0).max(0xffff).required())
   minorMask = 0xffff;
+  @(jf.number().integer().required())
   batteryMask = 0x00000000;
-  tagOverrides: { [key: string]: TagOverride } = {};
-
+  @(jf.object().custom(validateTagOverrides).required())
+  tagOverrides: { [entityId: string]: TagOverride } = {};
+  @(jf.number().integer().min(0))
   timeout = 60;
+  @(jf.number().integer().min(0))
   updateFrequency = 0;
+  @(jf.number().positive().optional())
   maxDistance?: number;
 }
+// TODO OR Properties(whitelist|allowlist)
 
-class TagOverride {
-  id?: string;
-  name?: string;
-  measuredPower?: number;
-  batteryMask?: number;
+const BluetoothLowEnergyScheme = jf
```

Comment:
Is this used somehow or just left over? If it is used by some joi magic then I think it would be a good idea to suppress the linter warning for this line, otherwise it can be removed imo.